src/ImuSensor.cc

```diff
@@ -14,17 +14,10 @@
  * limitations under the License.
  *
 */
-#ifdef _WIN32
-#pragma warning(push)
-#pragma warning(disable: 4005)
-#pragma warning(disable: 4251)
-#endif
-#include <gz/msgs/imu.pb.h>
-#ifdef _WIN32
-#pragma warning(pop)
-#endif
 
 #include <gz/common/Profiler.hh>
+#include <gz/msgs/imu.pb.h>
```

Comment:
Can we keep the warning suppressions for the message headers for now? I know we have an alternative fix for the warnings in the works, but right now this is bringing back some warnings that we had got rid of.

Reply:
Ah yes, I already had that applied in my workspace, so I didn't catch it.